angular/pxb-component-theme.scss

```diff
@@ -0,0 +1,51 @@
+// Import library functions for theme creation.
+@import '~@angular/material/theming';
+
+@mixin pxb-components-theme($theme) {
+    @include pxb-empty-state-theme($theme);
+    @include pxb-scorecard-theme($theme);
+    @include pxb-channel-value-theme($theme);
+    @include pxb-hero-theme($theme);
+}
+
+@mixin pxb-scorecard-theme($theme) {
+    $primary: map-get($theme, primary);
+    $foreground: map-get($theme, foreground);
+    .pxb-scorecard {
+        .pxb-scorecard-header {
+            background-color: mat-color($primary);
+            color: map-get($foreground, light);
+            .mat-card-subtitle {
+                color: inherit;
+            }
+        }
+    }
+}
+
+@mixin pxb-empty-state-theme($theme) {
+    $primary: map-get($theme, primary);
+    $foreground: map-get($theme, foreground);
+    .pxb-empty-state {
+        color: map-get($foreground, text);
```

Comment:
aren't any of these rules saying `color: map-get($foreground, text)` be redundant? Isn't the default color foreground text?

Reply:
We have to explicitly set the color because angular material doesnt apply the `text` foreground color on any parent element.  If I take the color off, the text color will be black. (#FFFFFF)